src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -581,31 +581,23 @@ export default class AircraftCommander {
             holdFixLocation = aircraft.position; // make a/c hold over their present position
             inboundHdg = aircraft.heading;
 
-            // TODO: these aren't `Waypoints` and they should be
             aircraft.fms.insertLegHere({
                 type: 'fix',
                 waypoints: [
-                    { // document the present position as the 'fix' we're holding over
-                        navmode: WAYPOINT_NAV_MODE.FIX,
-                        fix: '[custom]',
-                        location: holdFixLocation,
-                        altitude: aircraft.fms.altitudeForCurrentWaypoint(),
-                        speed: aircraft.fms.currentWaypoint.speed
-                    },
-                    { // Force the initial turn to outbound heading when entering the hold
+                    new Waypoint({
```

Comment:
for readability, this instantiation should be pulled out and assigned to a variable.